# Id del robot che sta inviando il messaggio
uint8    robot_id

# Id del waypoint di cui voglio conoscere la distanza
uint8    waypoint_id